bigtable/type_test.go

```diff
@@ -55,38 +109,29 @@ func TestStringProto(t *testing.T) {
 			},
 		},
 	}
-
-	got := StringType{}.proto()
+	st := StringType{Encoding: StringUtf8Encoding{}}
+	got := st.proto()
 	if !proto.Equal(got, want) {
 		t.Errorf("got type %v, want: %v", got, want)
 	}
-}
 
-func TestSumAggregateProto(t *testing.T) {
-	want := &btapb.Type{
-		Kind: &btapb.Type_AggregateType{
-			AggregateType: &btapb.Type_Aggregate{
-				InputType: &btapb.Type{
-					Kind: &btapb.Type_Int64Type{
-						Int64Type: &btapb.Type_Int64{
-							Encoding: &btapb.Type_Int64_Encoding{
-								Encoding: &btapb.Type_Int64_Encoding_BigEndianBytes_{
-									BigEndianBytes: &btapb.Type_Int64_Encoding_BigEndianBytes{},
-								},
-							},
-						},
-					},
-				},
-				Aggregator: &btapb.Type_Aggregate_Sum_{
-					Sum: &btapb.Type_Aggregate_Sum{},
-				},
-			},
-		},
+	gotJSON, err := MarshalJSON(st)
+	if err != nil {
+		t.Fatalf("Error calling ToJSON: %v", err)
 	}
-
-	got := AggregateType{Input: Int64Type{}, Aggregator: SumAggregator{}}.proto()
-	if !proto.Equal(got, want) {
-		t.Errorf("got type %v, want: %v", got, want)
+	wantJSON := "{\"stringType\":{\"encoding\":{\"utf8Raw\":{}}}}"
+	if strings.ReplaceAll(string(gotJSON), " ", "") != wantJSON {
```

Comment:
rather than testing that the json equals a known string, I think we can instead just test that round tripping the object from json and back yield the "correct" thing?  Comparing json strings has the potential for false errors.

Reply:
Done